src/neo/VM/Helper.cs

```diff
@@ -53,17 +53,7 @@ public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scrip
 
         public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scriptHash, string operation, CallFlags flags, params object[] args)
         {
-            if (args.Length == 0)
```

Comment:
In this related to this PR, @shargon?

Reply:
An optimization, reusing `EmitPush`

Reply:
I think this is not used in other place.

Reply:
If we want to call to this new syscall we will needed